lib/elm-test.js

```diff
@@ -444,3 +324,123 @@ ${testFileGlobs.join('\n')}
 Are the above patterns correct? Maybe try running elm-test with no arguments?
       `.trim();
 }
+
+function checkNodeJsVersion() {
```

Comment:
Do we need to do this? Will the `engines` field of in the package.json not warn users if their nodejs is too old. 

Just to be clear because the diff is misleading: this check is already done by elm-test just somewhere else.

Reply:
To try it out, I installed chalk using Node.js 8. Chalk also requires Node.js >= 10. This is what it looks like:

```
❯ node -v
v8.17.0

❯ npm i chalk
npm WARN notsup Unsupported engine for chalk@4.1.0: wanted: {"node":">=10"} (current: {"node":"8.17.0","npm":"6.13.4"})
npm WARN notsup Not compatible with your version of node/npm: chalk@4.1.0

+ chalk@4.1.0
added 1 package, updated 3 packages and audited 118 packages in 1.977s

4 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

So, a warning is printed but also kinda easy to miss.

But on the other hand, our current check doesn’t even get a chance to execute.

```
❯ node -v
v8.17.0

❯ ./bin/elm-test --help
/Users/lydell/forks/node-test-runner/lib/Parser.js:209
const lowerName = /^\p{Ll}[_\d\p{L}]*$/u;
                  ^

SyntaxError: Invalid regular expression: /^\p{Ll}[_\d\p{L}]*$/: Invalid escape
    at Object.<anonymous> (/Users/lydell/forks/node-test-runner/lib/Parser.js:209:19)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/lydell/forks/node-test-runner/lib/Runner.js:4:12)
    at Module._compile (module.js:653:30)
```

So we have to either restructure code so it works, or remove it.

I’m going to remove the check. Let me know if you want me to add it back.

Reply:
I think the npm warning is enough. Trying to maintain this check would be too hard I think and too brittle. I am not sad to see it go.